### PR TITLE
Coordinator/show name

### PR DESCRIPTION
**Pull request by evbambly**
image

### CODE REVIEW

File: packages/functions/.firebaserc [hunk 1]
```diff
@@ -1,5 +1,35 @@
 {
   "projects": {
-    "default": "blood-components"
+    "staging": "blood-components",
+    "default": "blood-components",
+    "production": "blood-components-9ad48"
+  },
+  "targets": {
```
Comment (yaron1m) [inline packages/functions/.firebaserc:7]: Please remove "targets" and leave only "projects"

File: packages/functions/src/coordinator/GetCoordinatorHandler.ts [hunk 2]
```diff
@@ -11,11 +12,15 @@ export default async function (
     console.error("Could not find calling user", callerId);
     throw Error(`User is not a coordinator`);
   }
+  const coordinatorDonorUser = await DonorDAL.getDonor(callerId);
```
Comment (yaron1m) [inline packages/functions/src/coordinator/GetCoordinatorHandler.ts:15]: Please run in parallel.

This means to call `DonorDAL.getDonor(callerId);` before you call `getCoordinator` but without the `await`, and only `await` on it later

Comment (yaron1m) [inline packages/functions/src/coordinator/GetCoordinatorHandler.ts:15]: Please run in parallel.

This means to call `DonorDAL.getDonor(callerId);` before you call `getCoordinator` but without the `await`, and only `await` on it later

File: packages/functions/src/testUtils/TestSamples.ts [hunk 3]
```diff
@@ -12,10 +12,11 @@ export const sampleUser = {
   testUser: false,
 };
 
-export function saveTestDonor(donorId: string) {
+export function saveTestDonor(donorId: string, donorFields?: Partial<DbDonor>) {
```
Comment (yaron1m) [inline packages/functions/src/testUtils/TestSamples.ts:15]: This already saves first and last names, why not use them?

File: packages/functions/src/coordinator/GetCoordinatorHandler.ts [hunk 4]
```diff
@@ -11,11 +12,15 @@ export default async function (
...
 
   return {
     coordinator: {
       role: coordinator.role,
       activeHospitalsForCoordinator: getCoordinatorHospitals(coordinator),
+      name: coordinatorDonorUser
```
Comment (yaron1m) [inline packages/functions/src/coordinator/GetCoordinatorHandler.ts:21]: Can this really be undefined?
Do we support coordinators who's donors have no names?

File: packages/coordinator/src/components/Header/CoordinatorHeader.tsx [hunk 5]
```diff
@@ -20,12 +21,14 @@ interface CoordinatorHeaderProps {
...
   flags: HeaderButtonFlags;
   getEmail: () => string | undefined;
+  coordinator: Coordinator | undefined;
 }
 
 export default function CoordinatorHeader({
   flags,
   onSignOut,
   getEmail,
+  coordinator,
```
Comment (yaron1m) [inline packages/coordinator/src/components/Header/CoordinatorHeader.tsx:31]: Can you add a story to CoordinatorHeader?

File: packages/functions/src/testUtils/TestSamples.ts (hunk 3 above)
Comment (evbambly) [reply packages/functions/src/testUtils/TestSamples.ts:15]: I want to expect() a test-specific variable

File: packages/functions/src/coordinator/GetCoordinatorHandler.ts (hunk 4 above)
Comment (evbambly) [reply packages/functions/src/coordinator/GetCoordinatorHandler.ts:21]: Safety?

Comment (yaron1m) [reply packages/functions/src/coordinator/GetCoordinatorHandler.ts:21]: I don't think this is needed, your call

File: packages/functions/src/testUtils/TestSamples.ts (hunk 3 above)
Comment (yaron1m) [reply packages/functions/src/testUtils/TestSamples.ts:15]: Umm... ok.

You can just expect the existing first and last name, but I don't think this is critical

File: packages/functions/src/coordinator/GetCoordinatorHandler.ts (hunk 2 above)
Comment (yaron1m) [reply packages/functions/src/coordinator/GetCoordinatorHandler.ts:15]: I see you marked as resolved, but I don't see any code changes

**Review by yaron1m — APPROVED** [review]

**Pull request merged**